examples/ads.amp.html

```diff
@@ -247,6 +248,12 @@ <h2 id="adman">Adman</h2>
       data-host="talos.adman.gr">
   </amp-ad>
 
+  <h2 id="admanmedia">AdmanMedia</h2>
+  <amp-ad width="300" height="250"
+          type="admanmedia"
```

Comment:
I'm not able to see a rendered example. Could you please check that the example ad is rendering?